README.md

```diff
@@ -72,12 +72,27 @@ make test
 
 The library files and dictionaries (`libedm4hep.so`, ...) must be put somewhere in `LD_LIBRARY_PATH`.
 
-
-
-
-
+## Python bindings
+There are python bindings for all the classes in the datamodel for working with
+the clases individually (for working with collections podio has its own
+bindings). Make sure that `install/lib` is in `LD_LIBRARY_PATH` and
+`install/python` is in `PYTHONPATH`:
```

Comment:
can you replace `install` with the proper CMake variable in there? just to be very explicit (and then you can add `(e.g. install/python`),

Reply:
Fixed